src/components/tabs/tabs.st.css

```diff
@@ -0,0 +1,130 @@
+@namespace "Tabs";
+
+:import {
+    -st-from: "../../project.st.css";
+    -st-named: borderDefault,
+               borderStyle,
+               color_Focus,
+               color_Disabled,
+               color_MainText_Background;
+}
+:import {
+    -st-from: "../selection-list/selection-list.st.css";
+    -st-default: SelectionList;
+}
+:import {
+    -st-from: "../selection-list/option.st.css";
+    -st-default: Option;
+}
+
+.root {
+    -st-states: rtl, horizontal-top, horizontal-bottom, vertical-before, vertical-after;
+}
+
+.root {
+    display: flex;
+    flex-direction: column;
+}
+
+.tabList {
+    -st-extends: SelectionList;
+}
+
+.tabList,
+.tabList:focused {
+    display: flex;
+    flex-grow: 0;
+
+    overflow-x: auto;
+    overflow-y: hidden;
+
+    width: 100%;
+
+    border: none;
+    border-bottom: value(borderDefault);
+
+    background: value(color_MainText_Background);
+}
+
+.tabList Option {
+    width: auto;
+    flex-shrink: 0;
+    padding: 18px;
+    border: none;
+    background: transparent;
+}
+
+.tabList Option:selected:hover,
+.tabList Option:selected {
+    color: value(color_Focus);
+
+    padding-bottom: 16px;
+    margin-bottom: 0px;
+
+    border: none;
+    border-bottom: value(borderStyle) 2px value(color_Focus);
```

Comment:
`value(borderStyle) 2px value(color_Focus)` repeats a lot, put it in local var.

Reply:
https://github.com/wix/stylable/issues/183